### PR TITLE
Improve --dump censor.

### PR DESCRIPTION
**Pull request by sebastienvercammen**
## Description
* Added some fields to censor.
* Return `<empty>` for censored fields that are empty.

## Motivation and Context
* Directories should always be censored.
* Helps to diagnose issues w/ empty fields (e.g. an empty `--status-page-password` = 404 on `/status`).

## How Has This Been Tested?
Untested.

## Types of changes
- [x] Enhancement

## Checklist:
- [x] My code follows the code style of this project.

### CODE REVIEW

**Comment by pogo-excalibur** [comment]
Is there any easy way to check if a value is custom (i.e. not set to it's default value) and highlight it?

**Comment by sebastienvercammen** [comment]
@pogo-excalibur Maybe by returning the parser itself and using its internal methods to determine the defaults. There's no actual "highlighting" in a text file, but we could add a custom tag.

But I would suggest that's something for a separate PR if anyone wants to pick that up.

**Comment by billyjbryant** [comment]
Should `shared_config` also be censored since it outputs the directory in the path? ``` 'shared_config': '/home/pogo/data/RocketMap/config/shared.ini',
```

File: pogom/utils.py [hunk 1]
```diff
@@ -1289,7 +1290,9 @@ def _censor_args_namespace(args, censored_tag):
         'status_name',
         'status_page_password',
         'hash_key',
-        'trusted_proxies'
+        'trusted_proxies',
+        'data_dir',
+        'locales_dir'
```
Comment (billyjbryant) [inline pogom/utils.py]: I would consider adding `shared_config`, everything else looks good :)

Comment (sebastienvercammen) [reply pogom/utils.py]: Good catch. 👍

**Pull request merged**